.github/workflows/ci.yml

```
@@ -0,0 +1,37 @@
+name: CI
+on: 
+  push:
+  pull_request:
+    branches:
+      - development
```

```suggestion
on: 
  push:
    branches:
      - development
      - master
  pull_request:
```
In its current incarnation the `push` hook will run on every push to every branch, every time. So the `pull_request` trigger on branches that propose a merge to `develop` is currently redundant. This is the reason that all pull requests are currently [registering duplicate CI runs](https://github.com/bcgov/digital_marketplace/pull/177/checks) for every configured job.

However, re-running tests on protected branches that can only be updated via pull request would have value in order to potentially identify a bad merge commit following merge of a pull request.

Furthermore, I would propose that we likely want tests to run on every pull request, regardless of its merge target (ie we care about a good merge from `develop` to `master` as much as we care about a good merge from `feature/${something}` to `develop`).

---

> As this PR has already been merged, please consider this peer review as subject for a future series of PRs addressing opportunity for continuous improvement and follow all existing team agreements for creation of new backlog issues, including development team review and prioritization of these backlog issues with the product owner @SteveChapmanBCDX

@wenzowski - Sorry, I'm not sure what you were trying to say here.  Are you recommending that your proposed change be reviewed and added to the backlog?

---

@SteveChapmanBCDX You got it! I'm proposing the development team review first, and that they make a recommendation to you if there's a development team consensus that doing so would create business value.

---

@wenzowski - Thanks - good plan!  I'll add it to the team's review list for Backlog Refinement - I booked meetings for Wednesday/Thursday this week.